modules/backup-etcd.adoc

```diff
@@ -51,7 +51,7 @@ The `cluster-backup.sh` script is maintained as a component of the etcd Cluster
 +
 [source,terminal]
 ----
-sh-4.4# /usr/local/bin/cluster-backup.sh /home/core/assets/backup
+sh-4.4# sudo /usr/local/bin/cluster-backup.sh /home/core/assets/backup
```

Comment:
doesn't sh-4.4# already denote being logged in as a superuser? Just a question--i'm sure that this is correct.

Reply:
Yes, it does -- this confused me in the original issue as well, but I assumed the reporter of the issue knew something that I didn't about accessing the host binaries after using `chroot /host` (which also requires root permissions to run within the debug shell if I understand it correctly).

I just tested this procedure myself both with and without `sudo`. It works in both cases, as `oc debug node/<name>` starts a debug shell with the user logged in as `root`.

@geliu2016, can you help me to understand why `sudo` is necessary here? Is it only required in certain circumstances?

Reply:
@kowen-rh as I know, cluster-backup.sh can only be executed by root user as I mentioned in bug: https://issues.redhat.com/browse/OCPBUGS-11614, and it have been fixed that this command could be run by non-root user, this is design change as I understanding.

Reply:
I think so.